core/geometry/src/test/serialization/IModelJson.test.ts

```diff
@@ -256,13 +263,10 @@ describe("CreateIModelJsonSamples", () => {
       for (const i of items) {
         const currFile = sourceDirectory + i;
         // skip known non-round-trip files ...
-        let numSkip = 0;
-        for (const candidate of skipList) {
-          if (currFile.lastIndexOf(candidate) >= 0)
-            numSkip++;
-        }
-        if (numSkip > 0)
-          continue;
+        let isFiltered = false;
+        for (const candidate of skipList)
+          if (currFile.lastIndexOf(candidate) >= 0) { isFiltered = true; break; }
+        if (isFiltered) continue;
```

Comment:
A bit surprise by the use of these one line ifs, are we now going with these ?

Reply:
No we just aren't enforcing this coding guideline via lint for whatever reason.

Reply:
Not fixing style guidelines stuff in tests as part of addon PR.